utils/dwarf.h

```diff
@@ -62,6 +63,15 @@ struct uftrace_dbg_info {
 	char *base_dir;
 };
 
+// arg struct for passing infos to and from build_dwarf_info_cb():
+struct dwarf_info_args {
+	struct uftrace_dbg_info *dinfo;
+	struct uftrace_symtab *symtab;
+	enum uftrace_pattern_type ptype;
+	struct strv *args;
+	struct strv *rets;
+};
```

Comment:
I think this is needed only if it has libdw dependency.  How about moving it to the .c file?